pkg/resource/rtestutils/assertions.go

```diff
@@ -0,0 +1,134 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package rtestutils
```

Comment:
Maybe something like `testresource`? Because right now some functions read like "resource test utils assert resource" which sounds a bit redundant.

Reply:
I thought `testresource` is too generic, even more weird `testresource.AssertResources`.

`testutils` is nice, but there might be multiple of them, so prefixing with `r` to make it unique

Reply:
`testresource.Assert` is good no?

Reply:
doesn't quite sound good to me, there might more similar functions, like getting a list of resource IDs. I moved only _some_ functions for now, let's see what will be needed